src/test/skript/tests/syntaxes/expressions/ExprRepeat.sk

```diff
@@ -0,0 +1,5 @@
+test "repeat expression":
+	assert "A" repeated 0 times is not set with "Repeat expression failed. Didn't return null with 0"
+	assert "B" repeated 3 times is "BBB" with "Repeat expression failed. Didn't return expected result expected 'BBB'"
+	set {_msg} to "C" repeated 3 times
+	assert length of {_msg} is 3 with "Repeat expression failed. Didn't return expected length, expected '3'"
```

Comment:
I'd like to see some more comprehensive tests. What about multi-letter strings? What about null strings, or null integers, or empty strings, or negative integers? That kind of stuff.

Reply:
Didn't see this was a review will include it in my next push

Reply:
If you wanna check my test and ensure they're up to par. Spent far too long on this and apparently the way repeat worked changed since I last remember

Reply:
Looks great, thanks!